design-system/packages/fields/package.json

```
@@ -14,7 +14,7 @@
     "@keystone-ui/popover": "^5.0.1",
     "date-fns": "^2.26.0",
     "react": "^17.0.2",
-    "react-day-picker": "npm:react-day-picker@^7.4.8",
+    "react-day-picker": "npm:react-day-picker@^8.0.4",
```

This was just written like this before for reasons that don't matter anymore
```suggestion
    "react-day-picker": "^8.0.4",
```